src/lint/validators.nim

```diff
@@ -149,6 +203,12 @@ proc isString*(data: JsonNode; key: string; path: Path; context: string;
                 &"The {format(context, key)} value is {q s}, but it must be " &
                  "a lowercase and kebab-case string"
             result.setFalseAndPrint(msg, path)
+        elif checkIsUuid:
```

Comment:
```suggestion
        # TODO: Enable the offline UUID checks after we've done a first-pass
        # to fix the UUIDs on the tracks
        elif false and checkIsUuid:
```

If we want to merge this PR before fixing the UUIDs on the tracks, we could do something like this to temporarily disable the check.